vsphere/setup.py

```diff
@@ -95,26 +55,23 @@ def find_version(*file_paths):
     packages=['datadog_checks.vsphere'],
 
     # Run-time dependencies
-    install_requires=list(set(runtime_reqs)),
+    install_requires=get_requirements('requirements.in')+[
```

Comment:
We need to add something to the CI to ensure `requirements.in` and `requirements.txt` do not diverge. Not necessarily in this PR, but let's keep it in mind.

Reply:
added a card to the backlog